helm-chart-sources/edge/values.yaml

```diff
@@ -0,0 +1,240 @@
+# Cribl Edge Helm Chart Values
+
+image:
+  repository: cribl/cribl
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 4.0.0
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Default environment variables for Cribl Edge Deployment
+env:
+  CRIBL_DIST_MODE: managed-edge
+  CRIBL_EDGE: 1
+# Downward API values
+envValueFrom:
+  - name: CRIBL_K8S_POD
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.name
+# Extra environment variables
+extraEnv: []
+# Extra environment variables (from Secrets)
+extraEnvFrom: []
+# Extra ConfigMaps
+extraConfigMaps: []
+# Extra Secrets
+extraSecretMounts: []
+# Extra Volume mounts
+extraVolumeMounts:
+  # This is needed to report Docker container info.
+  # Only Docker is supported today, containerd coming soon
+  - name: docker-sock
+    mountPath: /var/run/docker.sock
+    hostPath: /var/run/docker.sock
+  # This is needed to access host processes and metrics.
+  - name: host-root
+    mountPath: /hostfs
+    hostPath: /
+    readonly: true
+# Extra containers
+extraContainers: []
+# Extra init containers
+extraInitContainers: []
+# Additional Pod labels
+labels: {}
+
+cribl:
+  home: /opt/cribl
+  # if set, overrides cribl.leader value
+  existingSecretForLeader:
+  # CRIBL_DIST_MASTER_URL
+  leader:
+  # Bootstrap config
+  existingSecretForConfig:
+  config:
+  # cribl/local.yml:
+  #  foo: bar
+  readinessProbe:
+    httpGet:
+      path: /api/v1/health
+      port: 9420
+      scheme: HTTP
+    initialDelaySeconds: 15
+    timeoutSeconds: 1
+  livenessProbe:
+    httpGet:
+      path: /api/v1/health
+      port: 9420
+      scheme: HTTP
+    initialDelaySeconds: 15
+    timeoutSeconds: 1
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  # If create is false, and name is set, the DaemonSet attempts to use this name as the serviceAccountName
+  name: ""
+  automountServiceAccountToken: true
+
+# Cluster Roles
+rbac:
+  create: true
+  annotations: {}
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - configmaps
+        - endpoints
+        - limitranges
+        - namespaces
+        - nodes
+        - persistentvolumeclaims
+        - pods
+        - replicationcontrollers
+        - secrets
+        - services
+        - strategicMergePatches
+        - nodes/log
+        - nodes/metrics
+        - nodes/proxy
+        - nodes/spec
+        - nodes/stats
+      verbs: ['get', 'list', 'watch']
```

Comment:
I know we need `list` for all of these and `get` for some of them but we don't use `watch` at all currently. Dunno how tight we want to make this OOB.

Reply:
Looks like we need `get` for DaemonSet, Node, and Pod.

Reply:
True, but all verbs are read-only. I think that's what most folks will care about. We can definitively say the Service Account can't make any changes in the environment.